.github/workflows/dotnetcore.yml

```diff
@@ -71,3 +71,22 @@ jobs:
       run: dotnet nuget push out/*.nupkg -s https://www.myget.org/F/neo/api/v2/package -k ${MYGET_TOKEN} -ss https://www.myget.org/F/neo/symbols/api/v2/package -sk ${MYGET_TOKEN}
       env:
         MYGET_TOKEN: ${{ secrets.MYGET_TOKEN }}
+
+  Release:
+    #if: github.ref == 'refs/heads/master' && startsWith(github.repository, 'neo-project/')
+    #needs: Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Get version
+      run: |
+        sudo apt install libxml2-utils
+        find src -name *.csproj | xargs xmllint --xpath "//VersionPrefix/text() | //VersionSuffix/text()" | xargs printf "RELEASE_VERSION=v%s-%s" | xargs export
```

Comment:
xargs: export: No such file or directory

Reply:
@belane

Reply:
I've never used actions before 😅 
I think it fails because export is not a binary. We can try with `export $(...)` and build the version string with `xmlstarlet`.